app/src/androidTest/java/com/hedvig/app/feature/marketpicker/MarketPickerActivityTest.kt

```diff
@@ -0,0 +1,112 @@
+package com.hedvig.app.feature.marketpicker
+
+import android.content.Intent
+import androidx.preference.PreferenceManager
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.rule.ActivityTestRule
+import com.agoda.kakao.screen.Screen.Companion.onScreen
+import com.apollographql.apollo.api.toJson
+import com.hedvig.android.owldroid.graphql.GeoQuery
+import com.hedvig.app.R
+import com.hedvig.app.feature.marketpicker.screens.MarketPickerScreen
+import com.hedvig.app.feature.settings.Language
+import com.hedvig.app.feature.settings.SettingsActivity
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.hamcrest.CoreMatchers
+import org.junit.Assert
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class MarketPickerActivityTest {
+
+    @get:Rule
+    val activityRule = ActivityTestRule(MarketPickerActivity::class.java, false, false)
+
+    @Test
+    fun selectCorrectMarket() {
```

Comment:
This is nice!

This only tests one happy path, though. Should we add one or two tests going through a different happy path, and one through an unhappy path?